### PR TITLE
Change links to TRPL and add some Packt books

### PR DESCRIPTION
**Pull request by m4sk1n**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,12 @@ The main documentation is always the best beginning, so if you haven't read yet,
 * [Mastering Rust](https://www.packtpub.com/application-development/mastering-rust) - Vesa Kaihlavirta
 * :star: [Rust Anthology](https://github.com/brson/rust-anthology) - [Brian Anderson][]
 * :soon: [Rust in Action](https://www.manning.com/books/rust-in-action) - TS McNamara
-* :soon: [Network Programming in Rust](https://www.packtpub.com/application-development/network-programming-rust) - Abhishek Chanda
+* [Network Programming in Rust](https://www.packtpub.com/application-development/network-programming-rust) - Abhishek Chanda
+* [Rust Programming By Example](https://www.packtpub.com/application-development/rust-programming-example) -  Guillaume Gomez, Antoni Boucher
```
Comment (ctjhoa) [inline README.md]: This one is already tracked.

File: README.md [hunk 2]
```diff
@@ -69,7 +69,12 @@ The main documentation is always the best beginning, so if you haven't read yet,
...
+* [Learning Rust](https://www.packtpub.com/application-development/learning-rust) -  Paul Johnson, Vesa Kaihlavirta
+* [Rust Essentials](https://www.packtpub.com/application-development/rust-essentials-second-edition) -  Ivo Balbaert
+* [Rust Cookbook](https://www.packtpub.com/application-development/rust-cookbook) -  Vigneshwer Dhinakaran
+* [Mastering Rust](https://www.packtpub.com/application-development/mastering-rust) -  Vesa Kaihlavirta
```
Comment (ctjhoa) [inline README.md]: This one is already tracked.

File: README.md [hunk 3]
```diff
@@ -69,7 +69,12 @@ The main documentation is always the best beginning, so if you haven't read yet,
 * [Mastering Rust](https://www.packtpub.com/application-development/mastering-rust) - Vesa Kaihlavirta
 * :star: [Rust Anthology](https://github.com/brson/rust-anthology) - [Brian Anderson][]
 * :soon: [Rust in Action](https://www.manning.com/books/rust-in-action) - TS McNamara
-* :soon: [Network Programming in Rust](https://www.packtpub.com/application-development/network-programming-rust) - Abhishek Chanda
+* [Network Programming in Rust](https://www.packtpub.com/application-development/network-programming-rust) - Abhishek Chanda
+* [Rust Programming By Example](https://www.packtpub.com/application-development/rust-programming-example) -  Guillaume Gomez, Antoni Boucher
+* [Learning Rust](https://www.packtpub.com/application-development/learning-rust) -  Paul Johnson, Vesa Kaihlavirta
+* [Rust Essentials](https://www.packtpub.com/application-development/rust-essentials-second-edition) -  Ivo Balbaert
```
Comment (ctjhoa) [inline README.md]: We already have the first edition can you update it?

**Comment by ctjhoa** [comment]
Thanks for the contribution!

**Comment by m4sk1n** [comment]
@ctjhoa, is it ok now?

**Comment by ctjhoa** [comment]
Looks perfect! Thanks

**Pull request merged**